src/components/common/Smilies/index.js

```diff
@@ -0,0 +1,5 @@
+import './styles.scss';
```

Comment:
is this component needed?

Reply:
Removing it since it's only used in one place.